### DISCUSSION TITLE
Can I order completion items vertically?

### DISCUSSION THREAD

**Discussion by steplee** [nushell/nushell#10159] [Q&A]
With tab completion, can I get the shell to order them vertically rather than the horizontal default? It's harder for me to read the column-major orientation than the row-major one I'm used to.

Edit: While I'm here, is there a way to get a more compact "ls" output, utilizing multiple columns?

**Comment by fdncred** [comment]
I use this for a more compact `ls`
```nushell
def lsg [] { ls | sort-by type name -i | grid -c }
```

I'm not sure about changing the order of the completions.

**Reply by steplee** [reply]
Thanks, that's perfect
